src/Faker/Core/Uuid.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Specific reason strict_types was added here?

Reply:
No actual reason, its in my default file template. Though i think it is good that we add it on the new services, i will remove it for now